src/components/Status/index.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
-const determineFirstOnPage = (page, pageSize) => Math.max(page * pageSize, 0)
+const determineFirstOnPage = (page, pageSize) => Math.max(page * pageSize, 1)
```

Comment:
This seems wrong, why did you change it to a 1 starting index?

Reply:
from a user perspective, they expect to see 1-10, not 0-9